docker-compose-infra.yml

```diff
@@ -0,0 +1,88 @@
+# docker-compose.yml
+
+version: '3'
+services:
+  tenant_db:
+    image: postgres:15
+    ports:
+      - '5432:5432'
+    healthcheck:
+      test: [ "CMD-SHELL", "pg_isready", "-d", "postgres" ]
+      interval: 50s
+      timeout: 60s
+      retries: 5
+    environment:
+      POSTGRES_DB: postgres
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+      POSTGRES_PORT: 5432
+
+  pg_bouncer:
```

Comment:
switch to supavisor?

Reply:
I've implemented both:

- For single tenant uses pgBouncer for simplicity
- For multi-tenant uses Supavisor

However, supavisor can also be use as single tenant